### ISSUE TITLE
"Assessment in progress" dialog's text isn't announced by JAWS

### ISSUE THREAD

**Issue by dbjorge** [microsoft/accessibility-insights-web#309]
## Describe the bug

**To Reproduce**

1. Run JAWS
2. Go to an assessment's details view
3. Run the automated checks
4. close target page and details view
5. reopen target page and relaunch assessment

## Actual behavior

observe that JAWS announces the "Assessment in progress" dialog but not the content inside it

## Expected behavior

[WCAG](https://www.w3.org/TR/WCAG20-TECHS/ARIA18.html) says that dialogs should announce their text content (via describedby)

## Extension (please complete the following information)

- OS: Windows
- Version 1.200.29
- Chrome Version 72.0.3626.121
- Target Page any

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

[x] Did you search for similar issues in our github issues or on [stackoverflow](https://stackoverflow.com/questions/tagged/accessibility-insights)

## Additional context

Discovered while investigating bug 1458545
